assets/src/utils/common_utils.py

```diff
@@ -162,6 +162,25 @@ def __init__(self, file_bytes: bytes, file_name: str, ext: str) -> None:
         self.converted_file: Optional[bytes] = None
         self.converted_ext: Optional[str] = None
 
+    def convert_to_badgerdoc(self) -> bytes:
+        try:
+            converted_file = requests.post(
+                settings.convert_service_endpoint,
+                data=self.file_bytes,
+            )
+        except requests.exceptions.ConnectionError as e:
+            self.conversion_status = "conversion error"
+            logger_.error("Convert service connection error - detail: %s", e)
+            raise requests.exceptions.ConnectionError(e)
```

Comment:
Use `exception` logger so we preserve traceback as well (it's done automatically)
And could you use `raise ... from` statement to save traceback as well

Reply:
Assets' part was moved to separated PR